package.json

```diff
@@ -81,7 +82,6 @@
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^2.2.3",
     "eslint-plugin-react": "^6.6.0",
-    "faker": "^4.1.0",
```

Comment:
Yeah, and you're totally right, we'll probably remove `faker` in the 0.1 release. But for now its cool to move into deps